pkg/interface/soto/src/js/components/lib/sole.js

```diff
@@ -57,9 +57,11 @@ export class Share {
           switch (Object.keys(dex)[0]) {
             case 'del': if (at < dex.del) { dex.del++; } break;
             case 'ins': if ((at < dex.ins.at) ||
-                              ((at === dex.ins.at) && !(cha <= dex.ins.cha))) {
+                           ((at === dex.ins.at) && !(cha <= dex.ins.cha))) {
                 dex.ins.at++;
-              } break;
+              } else if (at >= dex.ins.at) dex.ins.at = at;  //NOTE possibly unpredictable behaviour
```

Comment:
please put this on a newline and within curly braces.

Reply:
Done.